src/connectors/webhook-auth-dom-inject.ts

```diff
@@ -0,0 +1,27 @@
+enum State {
+	Waiting = 'WAITING',
+	Rejected = 'REJECTED',
+	Accepted = 'ACCEPTED',
+	Confirmed = 'CONFIRMED',
+}
+
+declare interface Window {
+	wsState: () => State;
+	setWSState: (newState: State) => void;
+}
+
+setInterval(() => {
+	if (window.wsState() === State.Accepted) {
+		const search = new URLSearchParams(window.location.search);
+		window.postMessage(
+			{
+				sender: 'web-scrobbler',
+				type: 'confirmLogin',
+				applicationName: search.get('applicationName'),
+				userApiUrl: search.get('userApiUrl'),
```

Comment:
Do you think we should do any validation of input?  e.g. if it's a URL or do you think it's fine?

Reply:
I think it's fine. get is always a string, and the webhook handles any string. The user is also shown what the URL is when accepting.